addons/website_slides/static/src/js/systray_items/slide_channel_add_form.js

```diff
@@ -1,27 +1,20 @@
 /** @odoo-module **/
 
-import {NewContentFormController} from '@website/js/new_content_form';
-import viewRegistry from 'web.view_registry';
-import FormView from 'web.FormView';
+import {NewContentFormController, NewContentFormView} from '@website/js/new_content_form';
+import {registry} from "@web/core/registry";
 
-const AddCourseFormController = NewContentFormController.extend({
+export class AddCourseFormController extends NewContentFormController {
     /**
      * @override
      */
-    _getPath(state) {
-        return `/slides/${state.data.id}`;
-    },
-});
+    get path() {
+        return `/slides/${this.model.root.data.id}`;
```

Comment:
In fact, would not it be worth adding the website_url field in python for the model that miss it ? Would allow like 3 JS files less for a potential nice to have feature ? cc @rdeodoo (maybe to confirm with PDE)

Reply:
Update: now only `'forum.forum'` overrides `path()` getter!